content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,135 @@
+
+See [Formatting via environment variables](../../concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables) for more information.
+
+### NO_PROXY
+
+`NO_PROXY` is a comma-seperated list of hostnames, IP addresses or IP ranges that do not require a proxy.
+
+```terminal
+NO_PROXY=myhostname.com,10.11.12.0/16,172.30.0.0/16
+```
+
+### HTTP_PROXY
+
+`HTTP_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTP_PROXY=http://proxy.example.com
+```
+
+### HTTPS_PROXY
+
+`HTTPS_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTPS_PROXY=http://proxy.example.com
+```
+
+### PRISMA_HIDE_UPDATE_MESSAGE
+
+`PRISMA_HIDE_UPDATE_MESSAGE` is used to when checking if a Prisma update is cached. It's a truthy value.
+
+```js
+const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
+```
+
+### PRISMA_GENERATE_SKIP_AUTOINSTALL
+
+`PRISMA_GENERATE_SKIP_AUTOINSTALL` can be set to a truthy value to skip the auto-install of `prisma` CLI and `@prisma/client` dependencies (if they are missing) when using the `prisma generate` command.
+
+### SKIP_GENERATE
+
+`SKIP_GENERATE` can be set to a truthy value to skip running `prisma generate` on npm's postinstall lifecycle hook.
+
+### BROWSER
+
+`BROWSER` is for Prisma Studio to force which browser it should be open in, if not set it will open in the default browser. It's also used as a flag when starting the studio from the CLI.
+
+```js
+BROWSER=firefox prisma studio --port 5555
+```
+
+```terminal
+prisma studio --browser firefox
+```
+
+See [Studio](./command-reference/#studio) documentation for more information.
+
+### PRISMA_BINARIES_MIRROR
+
+`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN endpoint to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
+
+### PRISMA_MIGRATION_ENGINE_BINARY
+
+`PRISMA_MIGRATION_ENGINE_BINARY` is used to set a custom location for your own migration engine binary.
+
+```js
+PRISMA_MIGRATION_ENGINE_BINARY=custom/my-migration-engine-unix
+```
+
+### PRISMA_QUERY_ENGINE_BINARY
+
+`PRISMA_QUERY_ENGINE_BINARY` is used to set a custom location for your own query engine binary.
+
+```js
+PRISMA_QUERY_ENGINE_BINARY=custom/my-query-engine-unix
+```
+
+### PRISMA_INTROSPECTION_ENGINE_BINARY
+
+`PRISMA_INTROSPECTION_ENGINE_BINARY` is used to set a custom location for your own introspection engine binary.
+
+```js
+PRISMA_INTROSPECTION_ENGINE_BINARY=custom/my-introspection-engine-unix
+```
+
+### PRISMA_FMT_BINARY
+
+`PRISMA_FMT_BINARY` is used to set a custom location for your own format engine binary.
+
+```js
+PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
+```
+
+### QUERY_ENGINE_BINARY_PATH
+
+When connecting to the database the `QUERY_ENGINE_BINARY_PATH` is used to set a custom path to the binary.
```

Comment:
Where is this used exactly? Is this really something we want to document?

Reply:
Maybe not, I took it from a doc that @Jolg42 had put together. If its not public facing i can remove it. In fact, given that its kinda hidden away im guessing this is nothing that the public will be using anyway. Will remove